presidio-analyzer/presidio_analyzer/analyzer_engine.py

```diff
@@ -71,7 +71,10 @@ def Apply(self, request, context):
 
         entities = AnalyzerEngine.__convert_fields_to_entities(
             request.analyzeTemplate.fields)
+
+        logger.info("Getting language from request")
         language = AnalyzerEngine.get_language_from_request(request)
+        logger.info("Got language: '%s'", language)
```

Comment:
Better to use f"Got language {language}" or .format() as these don't fail when param is not a string

Reply:
fixed